app/digests/models.py

```diff
@@ -30,3 +42,13 @@ class Meta:
 
     def __str__(self):
         return f'{self.id}: {self.title}'
+
+
+@receiver(post_save, sender=Digest)
```

Comment:
checklist: this should happen after the database transaction has committed (otherwise downstream service receiving and loading the API may not see data that is still uncommitted)

Reply:
I have now switched to using an `transaction.on_commit()` callback to gurantee the transaction has completed before sending the event. Normally a signal is sufficient, but yes in this use case you are right.